.gitignore

```diff
@@ -27,6 +27,7 @@ src/web_interface/static/file_icons
 # Start scripts
 start_all_installed_fact_components
 start_fact_backend
+start_fact_database
 start_fact_db
```

Comment:
This line can be deleted, can't it?

Reply:
Definitely. I wasn't sure if you wanted to keep it since your install scripts still allow for the possibility of it being present when the db installation is run.